src/net_processing.cpp

```diff
@@ -4133,6 +4248,18 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         return;
     }
 
+    if (pfrom.IsPrivateBroadcastConn()) {
+        if (msg_type != NetMsgType::PONG) {
+            LogPrintLevel(BCLog::PRIVATE_BROADCAST,
+                          BCLog::Level::Debug,
+                          "Ignoring incoming message '%s', peer=%d%s\n",
+                          msg_type,
+                          pfrom.GetId(),
+                          fLogIPs ? strprintf(", peeraddr=%s", pfrom.addr.ToStringAddrPort()) : "");
+            return;
+        }
+    }
+
```

Comment:
b81c3cc19ec8055d04b38a72fdfabbfcb2ed2358

Why isn't this at the top of the super long function (and include `VERACK` also?)

Reply:
Because we want to ignore incoming messages after a successful handshake. Ignoring the `VERACK` would disturb the handshake.